src/tokens.ts

```diff
@@ -278,6 +279,19 @@ export interface IGitExtension extends IDisposable {
    * @param path Path from which the top Git repository needs to be found
    */
   showTopLevel(path: string): Promise<Git.IShowTopLevelResult>;
+
+  /**
+   * Ensure a.gitignore file
```

Comment:
```suggestion
   * Ensure a .gitignore file exists
```

Reply:
Thx for the review. Last commit implements this.